src/Distortion.cpp

```diff
@@ -265,14 +271,20 @@ void removeDistortion(double dx, double dy, double &ux, double &uy,
     case LROLROCNAC: {
 
       if (opticalDistCoeffs.size() != 1) {
-        throw "Distortion coefficients for LRO LROC NAC must be of size 1, current size: " +  std::to_string(opticalDistCoeffs.size());
+        csm::Error::ErrorType errorType = csm::Error::INDEX_OUT_OF_RANGE;
+        std::string message = "Distortion coefficients for LRO LROC NAC must be of size 1, current size: " +  std::to_string(opticalDistCoeffs.size());
+        std::string function = "removeDistortion";
+        throw csm::Error(errorType, message, function);
       }
 
       double dk1 = opticalDistCoeffs[0];
 
       double den = 1 + dk1 * dy * dy;     // r = dy*dy = distance from the focal plane center
       if (den == 0.0) {
-        throw "Unable to remove distortion for LRO LROC NAC. Focal plane position " + std::to_string(dy);
+        csm::Error::ErrorType errorType = csm::Error::ALGORITHM;
```

Comment:
Same as above, this seems to be the most appropriate error as it's a division by 0 error.